### DISCUSSION TITLE
Welcome to haskellcn Discussions!

### DISCUSSION THREAD

**Discussion by freizl** [HaskellCNOrg/haskellcn#178] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community instead of using Issues (which are all closed for reference purpose). We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Thank you all!

**Comment by xavierchow** [comment]
Is there any Haskell / Functional programming community in China?

**Reply by xiaolzha** [reply]
Haskell 中文 telegram channel：https://t.me/haskell_zh_2  
This channel is active.

**Reply by youwillbe** [reply]
我们有个微信群，300来人

**Reply by ymy12345769** [reply]
> 我們有微信群，300來人

你好请拉我谢谢你

**Reply by ymy12345769** [reply]
> 可以拉我進微信群嗎
你好我也想加入

**Reply by yangzai** [reply]
> 我们有个微信群，300来人

你好 我也想进群 谢谢
